packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -355,6 +363,7 @@ export abstract class InversifyHttpAdapter<
   ): void;
 
   protected abstract _buildRouter(
+    context: TContext,
     path: string,
     routerParams: RouterParams<TRequest, TResponse, TNextFunction>[],
     guardList: RequestHandler<TRequest, TResponse, TNextFunction>[] | undefined,
```

Comment:
5 params are maybe too many params, does it make sense to rename `RouterParams`to `RouteParams` or `EndpointParams` and extract some `ControllerParams` to cover `path`, `routerParams`, `guardList` and `middlewareList`?